seqr/views/apis/variant_search_api_tests.py

```diff
@@ -134,6 +135,17 @@ def test_query_variants(self, mock_get_variants, mock_get_gene_counts):
         results_model = VariantSearchResults.objects.get(search_hash=SEARCH_HASH)
         mock_get_variants.assert_called_with(results_model, sort='xpos', page=1, num_results=100)
 
+        # Test new search with allProjectFamilies
+        url1 = reverse(query_variants_handler, args=[SEARCH_HASH1])
```

Comment:
Can you make this a second test called something like `test_query_variants_all_projects`? This test is already getting very long and all the other tested  requests in it build off one another (as in, it is actually testing that seqr behaves correctly when you send multiple requests in a row for a single search which is a real user behavior as they do a search and then sort/page/download) while this is testing a completely different search activity

Reply:
I moved the test for the search with "allProjectFamilies" to a separated test function.